src/content/reference/react-dom/components/option.md

```diff
@@ -19,56 +20,56 @@ The [built-in browser `<option>` component](https://developer.mozilla.org/en-US/
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `<option>` {/*option*/}
 
-The [built-in browser `<option>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) lets you render an option inside a [`<select>`](/reference/react-dom/components/select) box.
+[Komponen `<option>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) memungkinkan Anda untuk me-*render* opsi di dalam kotak [`<select>`](/reference/react-dom/components/select).
 
 ```js
 <select>
-  <option value="someOption">Some option</option>
-  <option value="otherOption">Other option</option>
+  <option value="someOption">Sebuah opsi</option>
+  <option value="otherOption">Opsi lain</option>
 </select>
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh lebih banyak di bawah ini.](#usage)
 
 #### Props {/*props*/}
 
-`<option>` supports all [common element props.](/reference/react-dom/components/common#props)
+`<option>` mendukung semua [props element umum.](/reference/react-dom/components/common#props)
 
-Additionally, `<option>` supports these props:
+Sebagai tambahan, `<option>` mendukung *props* ini:
```

Comment:
```suggestion
Sebagai tambahan, `<option>` mendukung *props* berikut:
```

Because the original sentence is using "these" and there are some items below this.